lib/util.js

```diff
@@ -573,7 +585,8 @@ exports.isPrimitive = isPrimitive;
 function isBuffer(arg) {
   return arg instanceof Buffer;
 }
-exports.isBuffer = isBuffer;
+exports.isBuffer = deprecate(isBuffer,
+    'util.isBuffer is deprecated, please use a user-land alternative.');
```

Comment:
I'm not sure why `Buffer.isBuffer` was added initially. That'd be interesting to find out.

That said, `Buffer.isBuffer` has been incredibly useful to me in writing [`buffer`](https://github.com/feross/buffer) for browserify. The `Buffer` constructor actually returns a `Uint8Array` for performance reasons, but with all the buffer methods attached as properties. Works great, except for [one edge case](https://github.com/feross/buffer#important-always-use-bufferisbuffer-instead-of-instanceof-buffer) – `instanceof Buffer` doesn't work anymore. So, I've been recommending that people use `Buffer.isBuffer` and that's worked well. :-)

Eventually, I can just return a proper `Buffer` that's a subclass of `Uint8Array` and `instanceof Buffer` will start working :-)